core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/StackV4Endpoint.java

```diff
@@ -269,4 +273,17 @@ FlowIdentifier setClusterMaintenanceMode(@PathParam("workspaceId") Long workspac
     @ApiOperation(value = UPDATE_SALT, nickname = "updateSaltByName")
     FlowIdentifier updateSaltByName(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
 
+    @POST
```

Comment:
To match REST conventions and StackV4Endpoint naming conventions:
```suggestion
    @POST
    @Path("{name}/database_backup")
    @Produces(MediaType.APPLICATION_JSON)
    @ApiOperation(value = DATABASE_BACKUP, nickname = "databaseBackup")
    BackupV4Response backupDatabaseByName(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name,
            @QueryParam("location") String location, @QueryParam("backupId") String backupId);

    @POST
    @Path("{name}/database_restore")
    @Produces(MediaType.APPLICATION_JSON)
    @ApiOperation(value = DATABASE_RESTORE, nickname = "databaseRestore")
    RestoreV4Response restoreDatabaseByName(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name,
            @QueryParam("location") String location, @QueryParam("backupId") String backupId);
```

Reply:
Basically the suggestion is to accepted location and backup-id as query parameters, right.